tests/std/tests/P1023R0_constexpr_for_array_comparisons/test.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <array>
+#include <cassert>
+
+using namespace std;
+
+constexpr array<int, 5> a0{2, 8, 9, 1, 9};
+constexpr array<int, 3> a1{2, 8, 9};
+constexpr array<int, 5> a2{2, 8, 9, 1, 8};

```

Comment:
To avoid Clang warnings (which may no longer be an issue?) about brace elision, we conventionally use double braces to initialize `std::array`. That's because it's an aggregate (a struct) containing an aggregate (a built-in array). So:

```suggestion
constexpr array<int, 5> a0{{2, 8, 9, 1, 9}};
constexpr array<int, 3> a1{{2, 8, 9}};
constexpr array<int, 5> a2{{2, 8, 9, 1, 8}};
```

A single set of braces for the 0-length array is fine.

Reply:
Are you getting any warnings? I'm not (and checked to see if there was any way to enable more warnings in the documentation, but didn't find anything).

Reply:
I didn't actually try to build and run the changes. It's quite possible that Clang 9 became less aggressive about brace elision warnings compared to previous versions. Still, we use double braces in other tests, so I'd like to be consistent here.